plugin/EstateList.php

```diff
@@ -388,7 +388,12 @@ public function estateIterator($modifier = EstateViewFieldModifierTypes::MODIFIE
 			$pEstateStatusLabel = $this->_pEnvironment->getEstateStatusLabel();
 			$recordModified['vermarktungsstatus'] = $pEstateStatusLabel->getLabel($recordRaw);
 		}
-
+		if (isset($recordModified['multiParkingLot'])) {
+			$language = new Language();
+			$languageDefault = Language::getDefault();
+			$locate = $language->getLocale();
+			$recordModified['multiParkingLot'] = $this->formatParkingLot($recordModified['multiParkingLot'], $languageDefault, $locate);
```

Comment:
`formatParkingLot` is some functionality that needs to be available in the templates. `EstateList` should provide the template the raw parking lot data, not the formatted data.

Reply:
Hi @jayay . Because `$recordModified` is handled by `$pArrayContainer = new ArrayContainerEscape($recordModified)` the values in `multiParkingLot` if is Array will be converted to "Array" string

Reply:
That's bad. `ArrayContainerEscape` will only handle one level of arrays. Would it be a good solution to make `ArrayContainer(Escape)` to handle arbitrary levels of nested arrays?